Formula/libfreenect.rb

```diff
@@ -13,13 +13,33 @@ class Libfreenect < Formula
     sha256 "f714532e1b21365063746846544a340dac70cf0c5cc877a207dd17284ee100b7" => :mavericks
   end
 
+  option :universal
+  option "with-opencv", "Build with OpenCV support"
+  option "with-opencv3", "Build with OpenCV 3 support"
+  option "with-python", "Build with Python support"
+  option "with-python3", "Build with Python 3 support"
+
   depends_on "cmake" => :build
   depends_on "libusb"
+  depends_on "opencv" if build.with? "opencv"
+  depends_on "opencv3" if build.with? "opencv3"
+  depends_on "python" if build.with? "python"
+  depends_on "python3" if build.with? "python3"
```

Comment:
These should look like `depends_on :python3 => :optional`.

If this just builds extension modules, just always build the python 2 modules and don't declare a :python dependency.